.travis.yml

```diff
@@ -16,5 +16,7 @@ matrix:
       env: TOXENV=flake8
     - python: 2.7
       env: TOXENV=ansible-lint
+    - python: 2.7
+      env: TOXENV=ansible2-lint
```

Comment:
```
[testenv]
setenv=
    ANSIBLE_VERSION=""
basepython = python2.7
deps =
    -rtest-requirements.txt
    ansible{env:ANSIBLE_VERSION}

[testenv:ansible-lint]
setenv=
    ANSIBLE_VERSION=>=1.9.1,<2.0.0
commands =
    {toxinidir}/scripts/linting-ansible.sh
deps=
    {[testenv]deps}

[testenv:ansible2-lint]
setenv=
    ANSIBLE_VERSION=>=2.0
commands = {[testenv:ansible-lint]commands}
deps=
    {[testenv]deps}
```
Here's what it would look like if the ansible-lint jobs were completely 'templated'. My personal preference would be just to explicitly set the ansible version on it's own line because it's more readable; the current 1.9.1 to 2.0 range looks somewhat messy as an environment var to my eyes.